frontend/src/App.js

```diff
@@ -2,17 +2,21 @@ import LoginForm from './LoginForm';
 import React from "react";
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import SaveEmployee from './SaveEmployee';
+import SaveEmployee from './employees/SaveEmployee';
+import UpdateEmployee from './employees/UpdateEmployee';
 import './style.css';
 import TableListOfEmployees from './employees/TableListOfEmployees';
 
 export default function App() {
   return (
     <Router>
       <Switch>
-        <Route path='/addemployee'>
+        <Route path='/employees/new'>
           <SaveEmployee />
         </Route>
+        <Route path='/employees/update'>
+          <UpdateEmployee id='1' />
```

Comment:
Szerintem a routing megvalósításának is ebbe a pull requestbe kell tartoznia, mert ugye ez így nem lesz jó, hogy fixen az egyes munkatársat szerkesztjük. Prop helyett a szerkesztett munkatárs azonosítóját route paraméterből kellene venni és így nézne ki a route: /employees/1
Emlékeztető: https://reactrouter.com/web/example/url-params

Reply:
Igen, az az 1-es véletlenül maradt benne, köszi!!